src/managing-workflow/component-configuration.md

```diff
@@ -19,7 +19,6 @@ env:
     value: "admin_only"
 ```
 
-
```

Comment:
I think we need to find a markdown linter in the future if we want to enforce consistent styling. :)